admin/move_attachments.php

```diff
@@ -291,9 +295,13 @@ function move_attachments_to_disk( $p_type, array $p_projects ) {
 
 # Display results
 if( empty( $t_moved ) ) {
-	echo '<p class="lead">Nothing to do.</p>'. "\n";
+	echo '<div class="alert alert-danger">';
+		echo '<p class="lead"><strong>Opps!</strong> Please select the project you want to move the attachment.</p>';
```

Comment:
The old message indicated that there is no work to do and didn't mention anything about projects not selected.  Which can be the case of there were projects selected that had no attachments to be moved.  The new one seems to indicate that there is no project selected.  If we want the message to indicate that a project is selected, then we should look at project move list, otherwise, I would keep the old message to cover the case where there are no projects selected or projects selected, but no work to be done.